src/lib/homestore.cpp

```diff
@@ -234,27 +239,29 @@ void HomeStore::shutdown() {
     m_cp_mgr.reset();
 
     if (has_repl_data_service()) {
+        // Log and Data services are stopped by repl service
         s_cast< GenericReplService* >(m_repl_service.get())->stop();
+        m_log_service.reset();
```

Comment:
since we are making log service and data service inside repl service start, should we also make each one of them stop inside m_repl_service->stop() as well?